xgcm/duck_array_ops.py

```diff
@@ -5,6 +5,11 @@
 from __future__ import division
 from __future__ import print_function
 
+# fix for inconsistent numpy behavior
+import os
+
+os.environ["NUMPY_EXPERIMENTAL_ARRAY_FUNCTION"] = "1"
+
```

Comment:
I'm not really excited about this fix. We should not be modifying user environment variables. It's also not guaranteed to work; if the user imports numpy before importing xgcm, this won't have any effect.

Reply:
Good point. I just dont understand the internals of numpy enough. But let me remove these and see if the tests run (there is a good chance that requiring numpy >=1.16 solves this on its own.